contracts/managers/AssetPairManagerV2.sol

```diff
@@ -0,0 +1,402 @@
+/*
+    Copyright 2020 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.5.7;
+pragma experimental "ABIEncoderV2";
+
+import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import { ICore } from "set-protocol-contracts/contracts/core/interfaces/ICore.sol";
+import { ILiquidator } from "set-protocol-contracts/contracts/core/interfaces/ILiquidator.sol";
+import { IRebalancingSetTokenV3 } from "set-protocol-contracts/contracts/core/interfaces/IRebalancingSetTokenV3.sol";
+import { ISetToken } from "set-protocol-contracts/contracts/core/interfaces/ISetToken.sol";
+import { RebalancingLibrary } from "set-protocol-contracts/contracts/core/lib/RebalancingLibrary.sol";
+import { TimeLockUpgradeV2 } from "set-protocol-contracts/contracts/lib/TimeLockUpgradeV2.sol";
+
+import { IAllocator } from "./allocators/IAllocator.sol";
+import { ITrigger } from "./triggers/ITrigger.sol";
+
+
+/**
+ * @title AssetPairManagerV2
+ * @author Set Protocol
+ *
+ * Manager contract for implementing any trading pair and strategy for RebalancingSetTokenV3. Allocation 
+ * determinations are made based on output of Trigger contract. bullishBaseAssetAllocation amount is 
+ * passed in and used when bullish, allocationDenominator - bullishBaseAssetAllocation used when bearish.
+ * 
+ * CHANGELOG:
+ * - Support RebalancingSetTokenV3
+ * - Remove logic associated with pricing auctions, which has been moved to liquidator contracts
+ * - Add abilities to switch liquidator and liquidatorData, and ability to change fees
```

Comment:
We may also want to change the feeRecipient too

Reply:
Ah i see you did it. I suppose, just add it to the changelog then

Reply:
setFeeRecipient is already built in. Going to clarify this is the comment here